### PR TITLE
Changing hello to Good bye

### PR DESCRIPTION
**Pull request by R4NJI**
R4NJI is here

### CODE REVIEW

File: src/javaapplication/JavaApplication.java [hunk 1]
```diff
@@ -12,7 +12,7 @@ public class JavaApplication {
      */
     public static void main(String[] args) {
         // TODO code application logic here
-        System.out.print("hello world");
+        System.out.print("Good bye world");
```
Comment (leNigaud) [inline src/javaapplication/JavaApplication.java:15]: veloma eee

**Review by leNigaud — COMMENTED** [review]
veloma eee

File: src/javaapplication/JavaApplication.java [hunk 2]
```diff
@@ -12,7 +12,7 @@ public class JavaApplication {
      */
     public static void main(String[] args) {
         // TODO code application logic here
-        System.out.print("hello world");
```
Comment (leNigaud) [inline src/javaapplication/JavaApplication.java:15]: ty zao manahoana ee

File: src/javaapplication/JavaApplication.java [hunk 3]
```diff
@@ -12,7 +12,7 @@ public class JavaApplication {
      */
     public static void main(String[] args) {
```
Comment (leNigaud) [inline src/javaapplication/JavaApplication.java:13]: declaration de fonc
tion

**Review by leNigaud — APPROVED** [review]

File: src/javaapplication/JavaApplication.java (hunk 1 above)
Comment (RandrianantenainaBearisoa) [inline src/javaapplication/JavaApplication.java:15]: blabalbal

**Review by RandrianantenainaBearisoa — APPROVED** [review]

**Review by RandrianantenainaBearisoa — APPROVED** [review]

**Review by leNigaud — APPROVED** [review]

**Review by RandrianantenainaBearisoa — APPROVED** [review]

**Review by RandrianantenainaBearisoa — APPROVED** [review]
gang

**Pull request merged**